src/docker-compose

```diff
@@ -0,0 +1,318 @@
+#!/bin/sh
+# Copyright 2025 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Uncomment the line below to get a very verbose output from this Update Module.
+# set -x
+
+set -e
+
+STATE="$1"
+FILES="$2"
+TEMP_DIR="$FILES"/tmp
+
+JQ_CMD="jq"
+TAR_CMD="tar"
+DOCKER_CMD="docker"
+DOCKER_COMPOSE_CMD=""
+
+CONFIG_FILE="/etc/mender/mender-docker-compose.conf"
+PERSISTENT_STORE="/data/mender-docker-compose"
```

Comment:
In light of the purpose of this module, I wonder if we should instead use something like:

```suggestion
CONFIG_FILE="/etc/mender/mender-container-modules/docker-compose.conf"
PERSISTENT_STORE="/data/mender-container-modules/docker-compose"
```